packages/tools/src/utilities/stackPrefetch/stackPrefetch.ts

```diff
@@ -257,11 +257,11 @@ const updateToolState = (element) => {
 
   const minIndex = Math.max(
     0,
-    stack.currentImageIdIndex - configuration.minBefore
+    stack.currentImageIdIndex - (configuration.minBefore || 1)
   );
   const maxIndex = Math.min(
     stack.imageIds.length - 1,
-    stack.currentImageIdIndex + configuration.maxAfter
```

Comment:
how is it possible that the configuration is undefined? it is defined above. Why do mean || 5 and || 1 above

Reply:
If the user overrides the configuration but doesn't provide the values, then the || 5 or || 1 provide default values so that the add/subtract aren't undefined.